integration-tests/src/test/java/tachyon/master/MasterFaultToleranceIntegrationTest.java

```diff
@@ -70,11 +68,14 @@ private void faultTestDataCreation(TachyonURI folderName, List<Pair<Long, Tachyo
       throws IOException {
     mTfs.mkdirs(folderName);
     answer.add(new Pair<Long, TachyonURI>(mTfs.open(folderName).getFileId(), folderName));
-
+    // TODO why UnderStorageType.PERSIST will cause failure?
+    ClientOptions option = new ClientOptions.Builder(
+        mLocalTachyonClusterMultiMaster.getMasterTachyonConf()).setStorageTypes(
+        TachyonStorageType.STORE, UnderStorageType.NO_PERSIST).build();
     for (int k = 0; k < 10; k ++) {
       TachyonURI path =
-          new TachyonURI(PathUtils.concatPath(folderName, folderName.toString().substring(1), k));
-      mTfs.getOutStream(path, ClientOptions.defaults()).close();
+          new TachyonURI(PathUtils.concatPath(folderName, folderName.toString().substring(1) + k));
```

Comment:
Revert to original logic: https://github.com/amplab/tachyon/blob/branch-0.7/integration-tests/src/test/java/tachyon/master/MasterFaultToleranceIntegrationTest.java#L78